### PR TITLE
Update CONTRIBUTING.md

### LINKED ISSUE #453: Update CONTRIBUTING.md
Lots of things have changes since we initially wrote [CONTRIBUTING.md](./CONTRIBUTING.md).

This probably should be done on an ongoing basis.

### PR DESCRIPTION
**Pull request by amerameen**
Updated to the best of my knowledge. Please let me know if anything is still out of date.

### Issues: Closes #453

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
 # Contribution Guidelines
 
-Thanks for checking out our Polkadot Runtime Implementation! We're excited to hear and learn from you.
+Thank you for your interest on our implementation of the Polkadot Runtime Enviornment Implementation! We're excited to get to know you and work with you on gossamer. We've put together these guidelines to help you figure out how you can help us.
```
Comment (noot) [inline CONTRIBUTING.md]: interest in

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+
+The Web3 Foundation has a [Polkadot Wiki](https://wiki.polkadot.network/docs/en/learn-introduction) that would help both part-time and core contributors to the project in order to get up to speed. Our [wiki](https://github.com/ChainSafe/gossamer/wiki) also has some helpful resources. 
+
+The [Polkadot Runtime Specification](https://research.web3.foundation/en/latest/_static/pdfview/viewer.html?file=../pdf/polkadot_re_spec.pdf) serves as our primary specification, however it is currently in its draft status so things may be subject to change.
+
+One important thing distinction is that we are building the Polkadot Runtime Environment, not Polkadot itself. Given that, although a deep understanding of Polkadot is helpful, it's not critical to contribute to gossamer. To help understand how the Runtime Enviornment relates to Polkadot, check out this [talk that one of our team members gave at DotCon.](https://www.youtube.com/watch?v=nYkbYhM5Yfk)
```
Comment (noot) [inline CONTRIBUTING.md]: Environment

File: CONTRIBUTING.md (hunk 1 above)
Comment (amerameen) [reply CONTRIBUTING.md]: fixec

File: CONTRIBUTING.md (hunk 2 above)
Comment (amerameen) [reply CONTRIBUTING.md]: fixed this too! thanks for catching that

**Pull request merged**